presto-docs/src/main/sphinx/installation/jdbc.rst

```diff
@@ -15,6 +15,12 @@ The driver is also available from Maven Central:
         <version>\ |version|\ </version>
     </dependency>
 
+Requirements
+------------
+
+The JDBC driver is compatible with Java 8, and can be used with applications
+running on Java virtual machines version 8, 11 and higher.
```

Comment:
Let's change to

> running on Java versions 8 or higher.

Reply:
I used only the LTS releases since 9 and 10 are no longer supported by anyone at this stage, but changed to 8 or higher as requested.

Reply:
also left version (singular), plural sounded weird when reading the sentence